src/modules/Checkbox/Checkbox.jsx

```diff
@@ -72,7 +72,7 @@ export default {
           ref="input"
           class="hidden"
           readonly=""
-          tabiindex="0"
+          tabindex="0"
```

Comment:
Hey @mariolamacchia,
the tabindex attribute is required for the tab order (by this I mean focusing the checkbox when you press tab key again and again, see gif).

![2018-07-15 11 43 47](https://user-images.githubusercontent.com/1733706/42732682-f29e842a-8825-11e8-854b-d07f8321d204.gif)
(see https://codepen.io/amlang/pen/ajdppN)
The tabindex attribute is also "specified" by sematic UI (https://semantic-ui.com/modules/checkbox.html#/usage).
However, mdn advises against using a tab index greater than 1, as this can make navigation and operation of page content more difficult for people who depend on assistive technologies.

I hope I was able to answer your question.